#goal definition
int32 id
---
#result definition
bool completed
---
#feedback
string robot_status